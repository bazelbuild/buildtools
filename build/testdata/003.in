numbers = [
  0,
  11,
  123.456,
  123.,
  .456,
  1.23e45,
  -1,
  +1,
  0.0,
  -0.0,
  1.0,
  -1.0,
  +1.0,
  1e6,
  -1e6,
  -1.23e-45,
  3.539537889086625e+24,
  3.539537889086625E+24,
  3.539537889086625e00024000,
  3.539537889086625e+00024000,
  3539537889086624823140625,
  0x123,
  0xE45,
  0xe45,
]
