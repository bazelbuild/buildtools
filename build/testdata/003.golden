numbers = [
    0,
    11,
    123.456,
    123.,
    0.456,
    1.23e45,
    -1,
    +1,
    0.0,
    -0.0,
    1.0,
    -1.0,
    +1.0,
    1e6,
    -1e6,
    -1.23e-45,
    3.539537889086625e24,
    3.539537889086625e24,
    3.539537889086625e24000,
    3.539537889086625e24000,
    3539537889086624823140625,
    0x123,
    0xE45,
    0xe45,
]
